.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,98 @@
+name: "PR Build"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: setup-java-8
+        name: Setup Java 8
```

Comment:
We've been generally building with Java 11 (though if the build script isn't forcing Java 8 bytecode that would need to be fixed)

Reply:
Updated, and the target version is 8: https://github.com/open-telemetry/opentelemetry-java-contrib/blob/3de42160e92e0bd0ab876d835ee2273e11de959c/gradle/contrib.gradle#L10